foreman.fc

```diff
@@ -41,6 +41,8 @@
 /usr/lib/ruby/gems/1.8/gems/passenger-.*/helper-scripts/prespawn -- gen_context(system_u:object_r:passenger_exec_t,s0)
 /usr/share/passenger/helper-scripts/prespawn -- gen_context(system_u:object_r:passenger_exec_t,s0)
 
+/usr/lib6?4?/passenger/support-binaries/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
```

Comment:
Would `lib(64)?` also work and be cleaner?

Reply:
Sorry for the delay, yes definitely this is a simple regexp.